EIPS/eip-1576.md

```diff
@@ -0,0 +1,243 @@
+        },
+        [...] // (Optional) Other acknowledgement-related data
+      }
+    }
+
+**(5) Job details message**
+
+Bid requester responds to acknowledgement from bidder with any details required to execute on job. This will usually involve details on how the requester and bidder should proceed with an out of band communication channel. Relevant job details could also be shared over Whisper but users may run into message payload size limits.
+
+Topic: "topic" from (4)
+Encrypted by: "public_key" from (4)
+To: Bidder
+From: Bid requester
+
+    {
+      "session_uuid": "2c2c03eb-11b6-43e5-86fd-30ed8c13ad08", // A unique identifier 
+      "uuid": "abd8ab40-880f-4649-b3e3-d2c86ee4480d", // Message-specific unique-ID
+      "re_uuid": "80f8cec2-175b-46e2-99b4-8e9f836fc17a", // UUID of message being replied to
+      "type": "job_details", // Message type
+      "public_key": "...", // Single-user ECDSA public key for bidders to encrypt message for
+      "topic": "0xa4b3c2d1", 
+      "data": {
+        "url": "https://www....", // Endpoint for gathering information, providing information, or anything else
+        "job_data": {
+          [...] // Job data - entirely specific to type of job being performed, left to be specified per use case in the future
+        },
+        [...] // (Optional) Other acknowledgement-related data
+      } 
+    }
+
+**(6) Subsequent communications (optional)**
+
+Any subsequent communications naturally continue the key rotation and topic jumping patterns from the prior messages, retaining the relevant payload data schema, but carry no specified meaning in relation to the protocol.
+
+    {
+      "session_uuid": "2c2c03eb-11b6-43e5-86fd-30ed8c13ad08", // A unique identifier 
+      "uuid": "86bfc325-6d8a-4721-baf2-f18f0cde522a", // Message-specific unique-ID
+      "re_uuid": "abd8ab40-880f-4649-b3e3-d2c86ee4480d", // UUID of message being replied to
+      "type": "followup", // Message type, depends on implementation
+      "public_key": "...", // Single-user ECDSA public key for bidders to encrypt message for
+      "topic": "0x18273645", 
+      "data": {
+        [...] // Schema unspecified
+      } 
+    }
+
+## Implementation
+
+Both protocols are implemented (with some negligible schema changes) as the Bloom Protocol [Attestation Kit](https://github.com/hellobloom/attestation-kit).  The Whisper-specific implementation code can be found [here](https://github.com/hellobloom/attestation-kit/tree/master/app/shared/whisper), with Typescript specifications for message types found [here](https://github.com/hellobloom/attestation-kit/blob/master/app/shared/whisper/msgTypes.ts), and the majority of the sample code [here](https://github.com/hellobloom/attestation-kit/blob/master/app/shared/whisper/requesterActions.ts) and [here](https://github.com/hellobloom/attestation-kit/blob/master/app/shared/whisper/attesterActions.ts).
+
+
+## Copyright
+
+Copyright and related rights waived via CC0.
```

Comment:
You may want to add a link to the content of CC0
https://creativecommons.org/publicdomain/zero/1.0/